regulations/static/regulations/css/less/module/header.less

```diff
@@ -408,3 +408,20 @@ Small Screens
         padding-left: 4%;
     }
 }
+
+
+/*
+Wayfinding Overrides
+=====================
+In CFPB regs we sometimes have markerless paragraphs
+The markup contains these ID's so that we can link to them
+but we want to hide them in the wayfinding header.
+*/
+
```

Comment:
Flagging this for future... but I can't help thinking there's gotta be a better place to do this.